corehq/apps/users/models.py

```diff
@@ -2548,7 +2550,3 @@ def get(self, user_id):
             user = CouchUser.get_by_user_id(user_id)
             self.cache[user_id] = user
             return user
-
-
-from .signals import *
```

Comment:
I think the (very long time ago) purpose of this import was to make sure that all the `signal.connect` code fired whenever django bootstrapped. haven't read slack convo yet, but wanted to confirm that we were confident this change didn't accidentally change that behavior